app/components/personal_details/view.rb

```diff
@@ -57,6 +57,7 @@ def nationality_row
     end
 
     def nationality
+      return t("components.confirmation.not_provided_from_hesa_update") if nationalities.blank? && trainee.created_from_hesa
```

Comment:
I think all these checks need to be `trainee.hesa_record?` rather than `trainee.created_from_hesa`